.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+-   repo: https://github.com/psf/black
+    rev: 22.10.0
+    hooks:
+    -   id: black
```

Comment:
Isn't black too strict for us?

Current codebase uses no formatter as earlier developers were strong believers in making code look beautiful and readable manually.

So applying 'black' to all files changes a lot.

I think if any formatter should be forced, it is pep-8, because it fixes just most weird things, but leaves other be.
Though I am not sure it is a good idea to force any formatter in this repo at all because I don't currently see what problem we will solve with it.